pkg/network/cluster_config.go

```diff
@@ -104,7 +104,7 @@ func ValidateClusterConfig(clusterConfig configv1.NetworkSpec, client cnoclient.
 	// PlatformTypes, migration to DualStack is prohibited
 	if ipv4Service && ipv6Service || ipv4Cluster && ipv6Cluster {
 		if !isSupportedDualStackPlatform(infraRes.PlatformType) {
-			return errors.Errorf("DualStack deployments are allowed only for the BareMetal Platform type or the None Platform type")
+			return errors.Errorf("DualStack deployments are allowed only for the BareMetal Platform type or the vSphere Platform type or the None Platform type")
```

Comment:
Nit: Now that there are three platforms, it might be good to make this a comma-separated list. E.g. "DualStack deployments are allowed only for the BareMetal, vSphere, or None Platform types"

Reply:
sure, done.